src/main/java/org/sciborgs1155/robot/Intake/Intake.java

```diff
@@ -0,0 +1,49 @@
+package org.sciborgs1155.robot.Intake;
+
+import static org.sciborgs1155.robot.Intake.IntakeConstants.*;
+
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import monologue.Logged;
+import monologue.Monologue.LogBoth;
+import org.sciborgs1155.robot.Robot;
+
+public class Intake extends SubsystemBase implements Logged {
+
+  final PIDController pid = new PIDController(kp, kd, ki);
+
+  public final IntakeIO intake = Robot.isReal() ? new RealIntake() : new SimIntake();
+  @LogBoth public double targetSpeed = pid.getSetpoint();
+
+  @LogBoth
+  public double speed() {
+    return intake.getSpeed();
+  }
+
+  @Override
+  public void periodic() {
+    intake.setVoltage(pid.calculate(intake.getSpeed(), targetSpeed));
+  }
+
+  public Command intake() {
```

Comment:
does this work (like in sim and unit tests)? I'm worried it might not because it's just a runOnce command and there's no idle command afterwords, so in theory i think it should run it once and then go back to the default command, which would set the setpoint back to 0. But if it's working anyway for some reason, i guess that's fine